src/components/Callout/CalloutContent.tsx

```diff
@@ -130,6 +130,12 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
       this.props.onLayerMounted();
     }
 
+    // Set default directionalHint according to rtl setting.
+    // The is added here so that getRTL() is called on the client only.
+    if (!CalloutContent.defaultProps.directionalHint) {
```

Comment:
Ah! Yes, that is an option, that is probably the right way to do it, it will figure out the direction based on what side of the page it's on.